helmfile.d/stacks/monitoring.yaml

```diff
@@ -63,19 +63,21 @@ templates:
 
   openstack-monitoring:
     disableValidationOnInstall: true
-    condition: ck8sManagementCluster.enabled
-    installed: {{ .Values | get "openstackMonitoring.enabled" false }}
+    installed: {{ or (.Values | get "openstackMonitoring.enabled" false) (.Values | get "clusterApi.enabled" false) }}
```

Comment:
Did you consider if apps is being run on clusterAPI on a non-openstack provider?

Reply:
No, I had a discussion with Viktor and I should only keep `"openstackMonitoring.enabled" false` here
This value should apply for all openstack env, cluster-api and kubespray